src/opencolorio-1-fixes.patch

```diff
@@ -0,0 +1,83 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 13e2d64..47b5297 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -9,6 +9,12 @@ if(NOT DEFINED CMAKE_FIRST_RUN)
+     SET(CMAKE_FIRST_RUN 1 CACHE INTERNAL "")
+ endif()
+ 
++if (MSVC)
++    set(PATCH myptch)
++else()
++    set(PATCH patch)
++endif ()
++
+ ###############################################################################
+ ### GLOBAL ###
+ 
+@@ -171,12 +177,12 @@ else(USE_EXTERNAL_TINYXML)
+     endif()
+     ExternalProject_Add(tinyxml
+         URL ${CMAKE_SOURCE_DIR}/ext/tinyxml_${TINYXML_VERSION}.tar.gz
+-        PATCH_COMMAND patch -f -p1 < ${CMAKE_SOURCE_DIR}/ext/tinyxml_${TINYXML_VERSION}.patch
++        PATCH_COMMAND ${PATCH} --binary -f -p1 < ${CMAKE_SOURCE_DIR}/ext/tinyxml_${TINYXML_VERSION}.patch
```

Comment:
MXE installs tinyxml. Can opencolorio make use of system (i.e., MXE's) libraries?